.env.sample

```diff
@@ -0,0 +1,2 @@
+DEV_DB_USERNAME=
+DEV_DB_PASSWORD=
```

Comment:
@armpitude you'll need to create a `.env` file in the root directory, copy the stuff from `.env.sample` and add your db credentials that you'll create on MongoDB.